### DISCUSSION TITLE
Trouble generating a ssh key

### DISCUSSION THREAD

**Discussion by FrankC8174** [community/community#23853] [New to GitHub]
<p>Im having trouble creating a ssh key. when I opened gitbash and entered the phrase to generate the key per site instructions it errored say file not found. can any one help thanks in advance.</p>

**Comment by airtower-luna** [comment]
<p>You’ll need to provide some more details so people can help you, in particular the <em>exact</em> commands and error messages.</p>

**Comment by FrankC8174** [comment]
<p>my apologies! </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/4/d/4df2f821535f03f7cfb55a305334df0ec66a2da4.png" data-download-href="/uploads/short-url/b7zk7dW2QT5T633Dv2463aUNmo4.png?dl=1" title="Screenshot (7)" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/90303030/181079716-7b61b8b5-8ccc-4c3d-9886-d74206b36ac6.png" alt="Screenshot (7)" data-base62-sha1="b7zk7dW2QT5T633Dv2463aUNmo4" width="690" height="431" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/4/d/4df2f821535f03f7cfb55a305334df0ec66a2da4_2_690x431.png, https://aws1.discourse-cdn.com/github/optimized/3X/4/d/4df2f821535f03f7cfb55a305334df0ec66a2da4_2_1035x646.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/4/d/4df2f821535f03f7cfb55a305334df0ec66a2da4_2_1380x862.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/4/d/4df2f821535f03f7cfb55a305334df0ec66a2da4_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot (7)</span><span class="informations">1680×1050 83.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>ok I attempted to upload two screen shots 1 for the instructions I’m using and the second my terminal window.<br>
when I try to execute the command at step 4 it tells me no such file or directory. its probably something simple, however, I’m stuck any help would be appreciated</p>

**Comment by FrankC8174** [comment]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/4/3/436b53a465fb5b1da4a8cdd1dd291311b012b0af.png" data-download-href="/uploads/short-url/9CpW5IQkUmQYNThtOzS5rw75rfp.png?dl=1" title="Screenshot (6)" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/90303030/181079717-b8cd83ce-bded-46b6-9687-e8141ed72ad5.png" alt="Screenshot (6)" data-base62-sha1="9CpW5IQkUmQYNThtOzS5rw75rfp" width="690" height="431" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/4/3/436b53a465fb5b1da4a8cdd1dd291311b012b0af_2_690x431.png, https://aws1.discourse-cdn.com/github/optimized/3X/4/3/436b53a465fb5b1da4a8cdd1dd291311b012b0af_2_1035x646.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/4/3/436b53a465fb5b1da4a8cdd1dd291311b012b0af_2_1380x862.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/4/3/436b53a465fb5b1da4a8cdd1dd291311b012b0af_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot (6)</span><span class="informations">1680×1050 179 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by airtower-luna** [comment]
<p>The first screenshot says:</p>
<blockquote>
<p>Your public key has been saved in cs50.pub</p>
</blockquote>
<p>Looks like you told <code>ssh-keygen</code> to save the file under a non-standard name. That means you’ll have to use the name you chose (<code>cs50</code>) instead of the default (<code>id_rsa</code>) where applicable. :slightly_smiling_face:</p>
<p>Mind that <code>.pub</code> denotes the public key, don’t upload the private one!</p>

**Comment by FrankC8174** [comment]
<p>I really really appreciate your time and understanding. ok so I did try to use the cs50 I saved it under, but it still said file not found, so I deleted the file and re generated another key and saved it under the file name given in the instructions. then tried to execute the command using the file name id_rsa and it still said file not found.</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/0/4/047d65751c5655449baf226721316012e41683c9.png" data-download-href="/uploads/short-url/DIzm3CwcbPAVvfVAztD6HvHKfT.png?dl=1" title="Screenshot (10)" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/90303030/181079720-c0bb09ca-bb0a-4568-b736-84e3489096bd.png" alt="Screenshot (10)" data-base62-sha1="DIzm3CwcbPAVvfVAztD6HvHKfT" width="690" height="431" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/0/4/047d65751c5655449baf226721316012e41683c9_2_690x431.png, https://aws1.discourse-cdn.com/github/optimized/3X/0/4/047d65751c5655449baf226721316012e41683c9_2_1035x646.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/0/4/047d65751c5655449baf226721316012e41683c9_2_1380x862.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/0/4/047d65751c5655449baf226721316012e41683c9_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot (10)</span><span class="informations">1680×1050 96.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by FrankC8174** [comment, marked as answer]
<p>i got it thanks for your help</p>
